### PR TITLE
bat: add more information link

### PR DESCRIPTION
**Pull request by agucova**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by agucova** [comment]
Not sure how I screwed up the branches, I made the pull request directly from my fork to master.

File: pages.es/common/bat.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# bat
+
+> Imprime y concatena archivos.
+> Un clon de `cat` con resaltado de sintaxis e integración con `git`.
```
Comment (navarroaxel) [inline pages.es/common/bat.md:4]: Usamos `Git` porque no hablamos del comando sino del VCS en general.
```suggestion
> Un clon de `cat` con resaltado de sintaxis e integración con Git.
```

File: pages.es/common/bat.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Imprimir los contenidos de un archivo a la salida estándar (`stdout`):
+
+`bat {{archivo}}`
+
+- Concatenar varios archivos creando un nuevo archivo:
```
Comment (navarroaxel) [inline pages.es/common/bat.md:10]:
```suggestion
- Concatena varios archivos creando un nuevo archivo:
```

File: pages.es/common/bat.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# bat
+
+> Imprime y concatena archivos.
+> Un clon de `cat` con resaltado de sintaxis e integración con `git`.
+
+- Imprimir los contenidos de un archivo a la salida estándar (`stdout`):
```
Comment (navarroaxel) [inline pages.es/common/bat.md:6]: Comunmente usamos `salida estándar` o `stdout` como sinonimos, no los relacionamos porque suponemos que el usuario ya sabe.
```suggestion
- Imprime los contenidos de un archivo a la salida estándar:
```

File: pages.es/common/bat.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`bat {{archivo}}`
+
+- Concatenar varios archivos creando un nuevo archivo:
+
+`bat {{archivo1}} {{archivo2}} > {{archivo_objetivo}}`
```
Comment (navarroaxel) [inline pages.es/common/bat.md:12]: `archivo_objetivo` es una traducción muy literal, y no se usa mucho en español. Mejor usar `archivo_resultado` o `archivo_final`:
```suggestion
`bat {{archivo1}} {{archivo2}} > {{archivo_final}}`
```

File: pages.es/common/bat.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Añade múltiples archivos al final de un archivo objetivo:
+
+`bat {{archivo1}} {{archivo2}} >> {{archivo_objetivo}}`
```
Comment (navarroaxel) [inline pages.es/common/bat.md:16]:
```suggestion
`bat {{archivo1}} {{archivo2}} >> {{archivo_final}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @agucova, welcome and thanks for your submission. I left some comments.

**Comment by agucova** [comment]
Thanks for the suggestions, I'll keep a closer eye on future translations.

File: pages.es/common/bat.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`bat {{archivo1}} {{archivo2}} > {{archivo_final}}`
+
+- Añade múltiples archivos al final de un archivo objetivo:
+
+`bat {{archivo1}} {{archivo2}} >> {{archivo_final}}`
+
+- Numera las lineas del archivo:
+
+`bat -n {{file}}`
```
Comment (navarroaxel) [inline pages.es/common/bat.md:20]:
```suggestion
`bat -n {{archivo}}`
```

File: pages.es/common/bat.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Muestra un archivo JSON con resaltado de sintaxis:
+
+`bat --language json {{file.json}}`
```
Comment (navarroaxel) [inline pages.es/common/bat.md:24]:
```suggestion
`bat --language json {{archivo.json}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by ferrvic** [comment]
For the missing homepage you could use the link to the main repository here, in github: https://github.com/sharkdp/bat

**Comment by agucova** [comment]
> For the missing homepage you could use the link to the main repository here, in github: [sharkdp/bat](https://github.com/sharkdp/bat)

I was thinking of doing that, but since the original English translation didn't include it either, I thought it was necessary to first modify it there. (Keeping them in sync)

I'm not sure if that's the case.

**Comment by ferrvic** [comment]
> > For the missing homepage you could use the link to the main repository here, in github: [sharkdp/bat](https://github.com/sharkdp/bat)
> 
> I was thinking of doing that, but since the original English translation didn't include it either, I thought it was necessary to first modify it there. (Keeping them in sync)
> 
> I'm not sure if that's the case.

Well, I'm not sure what is the policy of the mantainers on this cases. Anyway, if you want, you could always submit another patch for the english page to add the project web page.

**Comment by agucova** [comment]
I've added it, pending my PR to the english page.

**Comment by navarroaxel** [comment]
@agucova You can add/edit an English page (and only 1) and include the translations in the same PR. So, you can add the edition for the English page in this PR and include the Spanish translation.
And why not add the link for the pages in `it`, `de`, `ko` and `pt_BR`.

**Comment by navarroaxel** [comment]
LGTM

**Review by sbrl — APPROVED** [review]

**Pull request merged**